[bn] Localize cloud-controller-manager.md

AhVir opened the pull request:
Localization of **cloud-controller-manager.md**
Related issue : #45729

AhVir committed:
Create cloud-controller-manager.md

AhVir committed:
Update cloud-controller-manager.md

sajibAdhi reviewed (commented):
Hey @AhVir 
Thanks for this great contribution! I've reviewed the changes and have a few suggestions that I think will improve the code's readability.

Kindly update your branch from upstream [branch](https://github.com/kubernetes/website/tree/dev-1.31-bn.1) when [this PR](https://github.com/kubernetes/website/pull/52000) is merged to fix the Netlify build [issue](https://app.netlify.com/projects/kubernetes-io-main-staging/deploys/689fb9f52e17ea0008b83e25).

sajibAdhi commented:
Hey, @AhVir
Kindly update your branch from upstream [branch](https://github.com/kubernetes/website/tree/dev-1.31-bn.1) and update the code as suggested.

AhVir commented:
/remove-lifecycle stale

AhVir commented:
Hi @sajibAdhi, thanks for the review and suggestions. I’ve updated the branch from upstream and addressed both of the requested translation changes.

The PR is ready for another review. Thank you!
